app/views/foreman_discovery/debian_kexec.erb

```diff
@@ -29,12 +32,31 @@ Extra options like --reset-vga can be set via "extra" array.
   options << @host.facts['append']
   options << "domain=#{@host.domain}"
   options << 'console-setup/ask_detect=false console-setup/layout=USA console-setup/variant=USA keyboard-configuration/layoutcode=us localechooser/translation/warn-light=true localechooser/translation/warn-severe=true'
-  options << "locale=#{host_param('lang') || 'en_US'}"
+  options << "locale=#{@host.params['lang'] || 'en_US'}"
```

Comment:
Please use the `host_param` macro, this has been deprecated already. Are you on some older version?